package.json

```diff
@@ -56,11 +56,11 @@
     "d3-time-format": "^4.1.0",
     "gray-matter": "^4.0.3",
     "lodash": "^4.17.21",
-    "next": "^12.1.6",
+    "next": "12.3.0",
     "next-mdx-remote": "^4.0.3",
     "prism-themes": "^1.9.0",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
```

Comment:
Latest react 🔥 😄

Reply:
Woohoo!!! I'll have to read about the [new features](https://reactjs.org/blog/2022/03/29/react-v18.html). Anything you've found extra interesting or helpful in react 18?